apps/finance/app/src/components/BalanceToken.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react'
-import { GU, textStyle, useTheme } from '@aragon/ui'
+import { GU, formatNumber, textStyle, useTheme } from '@aragon/ui'
 import { useNetwork } from '@aragon/api-react'
 import { tokenIconUrl } from '../lib/icon-utils'
-import { formatTokenAmount } from '../lib/utils'
```

Comment:
Wondering if this is still in use in other parts of the finance app; if so, we should also use the implementation from `aragonUI`.